mne/decoding/csp.py

```diff
@@ -602,6 +603,12 @@ def _concat_cov(self, x_class, *, cov_kind, log_rank):
 
     def _epoch_cov(self, x_class, *, cov_kind, log_rank):
         """Mean of per-epoch covariances."""
+        name = self.reg if isinstance(self.reg, str) else "empirical"
+        name += " with shrinkage" if isinstance(self.reg, float) else ""
```

Comment:
usually the name would be inferred using this function: https://github.com/mne-tools/mne-python/blob/dda454d7e4bd2766a2bdff37c06583fde99ba747/mne/cov.py#L769

here: https://github.com/mne-tools/mne-python/blob/dda454d7e4bd2766a2bdff37c06583fde99ba747/mne/cov.py#L2146-L2147

and that check is of course still performed ... it is just 2 levels deeper, so I did not see an inexpensive way to obtain that log information.

Perhaps what I did here is sufficient.